src/components/StyledMarkdown.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import {
+  Dimensions,
+  Linking,
+  Platform,
+  WebView,
+} from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+import defaultHTML from '../utils/default-md-html'
+
+import { removeMDTab } from '../utils/remove-md-tab'
+
+const MarkdownIt = require('markdown-it'),
+    md = new MarkdownIt({ linkify: true, breaks: true }).use(removeMDTab)
+
+const WEBVIEW_REF = 'WEBVIEW_REF'
+
+class StyledMarkdown extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { height: 0 }
+  }
+
+  onShouldStartLoadWithRequest = (event) => {
+    if (event.url.indexOf('http') >= 0 ) {
+      this.refs[WEBVIEW_REF].stopLoading()
+      Linking.openURL(event.url)
+      return false
+    } else {
+      return true
+    }
+  }
+
+  onNavigationStateChange = (navState) => {
+    if (Platform.OS === 'android') {
+      this.onShouldStartLoadWithRequest(navState)
+    }
+    if (Number(navState.title) > 0) {
+      const htmlHeight = Number(navState.title) //convert to number
+      this.setState({height: htmlHeight})
+      this.props.onReceivedHeight(htmlHeight)
+    }
+  }
+
+  render() {
+    //Markdownz uses markdown-it-imsize for image sizes, however this package
+    //requires fs, which is a node core module and not available natively
+    //Also, we can't use the image sizes.  This regexp removes them, prior to
+    //being sent to renderer, or else they're interpreted as links
+    let markdown = this.props.markdown.replace(/ =[0-9]+x[0-9]*\)/g, '\)')
```

Comment:
Well, I looked at the code being used by markdownz that does the image sizes (https://github.com/edpaget/markdown-it-imsize/blob/master/lib/index.js) and constructed a regexp based on the criteria they use.  (I couldn't use markdownz because it didn't work with react-native babel so I basically tried to reproduce it as much as possible, with markdown-it).  
 I couldn't do this as a markdown plugin because it gets parsed first and was already being turned into a link by the time it made it to the plugin (but since these are images, we want to display the image and not the link).  I can add it to the comments of this file if you think that would be best.